### PR TITLE
Import coala.io contributors functionality

### PR DESCRIPTION
**Pull request by sks444**
Closes https://github.com/coala/community/issues/32

### CODE REVIEW

File: data/management/commands/import_contributors_data.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                reviews=reviews
+            )
+            if created:
+                c.save()
+                display_format = "\nContributor, {}, has been saved."
+                print(display_format.format(c))
+        except Exception as ex:
+            print(str(ex))
+            msg = "\n\nSomething went wrong saving this contributor: {}\n{}".format(login, str(ex))
+            print(msg)
```
Comment (andrewda) [inline data/management/commands/import_contributors_data.py]: Any reason to save this in a `msg` variable first before printing? Just dilutes the code in my opinion.

File: data/management/commands/import_contributors_data.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                name=name,
+                bio=bio,
+                num_commits=num_commits,
+                issues_opened=issues_opened,
+                reviews=reviews
+            )
+            if created:
+                c.save()
+                display_format = "\nContributor, {}, has been saved."
+                print(display_format.format(c))
```
Comment (andrewda) [inline data/management/commands/import_contributors_data.py]: Same as below - probably don't need to save the message in a variable first.

File: data/management/commands/import_contributors_data.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        except Exception as ex:
+            print(str(ex))
```
Comment (andrewda) [inline data/management/commands/import_contributors_data.py]: We're printing the exception below, right? No need to print it here.

File: data/management/commands/import_contributors_data.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import requests
+import json
+from data.models import Contributor
+from django.core.management.base import BaseCommand
+
+IMPORT_URL = 'https://webservices.coala.io/contrib/'
```
[the remark this answers is not in the document]
Comment (jayvdb) [reply data/management/commands/import_contributors_data.py]: this needs to be moved to an environment variable, so that it is not coala specific.

**Comment by jayvdb** [comment]
The data import Command needs to be run from Travis

File: community/urls.py [hunk 5]
```diff
@@ -63,4 +64,6 @@ def get_index():
         distill_func=get_index,
         distill_file='log/index.html',
     ),
+    url(r'^data/', include('data.urls')),
```
Comment (jayvdb) [inline community/urls.py]: you need to use `distill_url`  ;-)

Comment (sks444) [reply community/urls.py]: damn! never used that, let me read their docs then :)

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**